ftw/simplelayout/aliasblock/browser/aliasblock.py

```diff
@@ -1,7 +1,33 @@
 from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
 from ftw.simplelayout.browser.blocks.base import BaseBlock
+from ftw.simplelayout.interfaces import ISimplelayoutBlock
+from zExceptions import Unauthorized
+
+
+try:
+    from ftw.slider.interfaces import IContainer
+except ImportError:
+    class IContainer(object):
+        """Dummy class if ftw.slider isn't installed.
+        """
 
 
 class AliasBlockView(BaseBlock):
 
     template = ViewPageTemplateFile('templates/aliasblock.pt')
+
+    def __init__(self, context, request):
+        super(AliasBlockView, self).__init__(context, request)
+        self.referenced_page = self.context.alias.to_object
+
+    def get_referenced_block_content(self):
+        """Returns the rendered simplayout block"""
+        try:
```

Comment:
In order to get the right `block_view` you should use the generic approach from https://github.com/4teamwork/ftw.simplelayout/blob/430e8e20e8bb84122b4e3e284e17553c51fce5ce/ftw/simplelayout/tests/test_multiview_block_properties.py